guides/common/modules/con_how-to-synchronize-content-using-export-and-import.adoc

```diff
@@ -13,3 +13,11 @@ For more information, see xref:Using_Upstream_Server_to_Sync_Content_View_Versio
 * You sync a single repository.
 This can be useful if you use the Content-View syncing approach, but you want to sync an additional repository without adding it to an existing Content View.
 For more information, see xref:synchronizing-a-single-repository_{context}[].
++
+[NOTE]
+====
+When you are unable to match upstream and downstream {Project} versions, you can use:
```

Comment:
I still feel like it needs the additional context -- if not the why, at least the requirement itself:

```suggestion
Synchronizing content using export and import requires the same major, minor, and patch version of {Project} on both the downstream and upstream {ProjectServer}s.

When you are unable to match upstream and downstream {Project} versions, you can use:
```